### PR TITLE
Geo: Add .geojson file with GEE forest extent estimations 

### PR DESCRIPTION
**Pull request by eraviolo**
To import the .json data into the forest-estimations table run:
`yarn geo:forest-estimations-import "FRA_236_ISO_v9.geojson"`

### CODE REVIEW

File: tools/geo/forestEstimationsImport.ts [hunk 1]
```diff
@@ -20,21 +20,30 @@ export const forestIndicatorsImport = async (): Promise<void> => {
   // eslint-disable-next-line no-console
   const args = process.argv.slice(2)
 
-  const path = args[0]
-  const content = await fs.readFile(path)
+  const fileName = args[0]
+
+  const content = await fs.readFile(path.resolve(__dirname, fileName))
```
Comment (minotogna) [inline tools/geo/forestEstimationsImport.ts]: can this just be 
```
  const content = await fs.readFile(path.resolve(__dirname, 'FRA....json'))
```
or there is a need to keep it dynamic ?

Comment (eraviolo) [reply tools/geo/forestEstimationsImport.ts]: Sure, we can omit the name, no need to keep it dynamic

Comment (minotogna) [reply tools/geo/forestEstimationsImport.ts]: cool :D so anyone can just run the script, without worrying about file name to pass

**Review by sorja — APPROVED** [review]
👍

**Review by minotogna — APPROVED** [review]

**Pull request merged**